src/atoms/Button.jsx

```diff
@@ -1,5 +1,6 @@
+import "../styles/button.css"
 const Button = () => {
```

Comment:
Aqui el label debes de agregarlo como si fuera un prop 
```suggestion
import "../styles/button.css"
const Button = ({ label }) => {
```

https://legacy.reactjs.org/docs/components-and-props.html